runtime/src/main/java/com/flipkart/flux/guice/interceptor/TransactionInterceptor.java

```diff
@@ -54,30 +73,22 @@ public Object invoke(MethodInvocation invocation) throws Throwable {
 
             if (transaction != null) {
                 transaction.commit();
```

Comment:
According to your example - method1() calling method2(),
Won't this commit the tx after method 2?